chatmaild/src/chatmaild/filtermail.py

```diff
@@ -101,13 +101,12 @@ def check_armored_payload(payload: str, outgoing: bool):
         return False
     payload = payload.removesuffix(suffix)
 
-    # Disallow comments in outgoing messages
     version_comment = "Version: "
     if payload.startswith(version_comment):
-        version_line = payload.splitlines()[0]
-        payload = payload.removeprefix(version_line)
-        if outgoing:
+        if outgoing:  # Disallow comments in outgoing messages
             return False
+        # Remove comments from incoming messages
+        payload = payload.split("\r\n", maxsplit=1)[1]
 
     while payload.startswith("\r\n"):
         payload = payload.removeprefix("\r\n")
```

Comment:
btw, it is not clear to me if this is really needed? in https://github.com/chatmail/relay/pull/655 it was introduced and there only the version prefix was removed leaving the `\r\n` and hence this was needed, but can it really happen that there are more than one `\r\n` after `Version: ProtonMail\r\n` ??

Reply:
It's slightly more robust, but computationally inexpensive, so I'd rather keep it.